string-manipulation-1/src/get-first-char.js

```diff
@@ -0,0 +1,24 @@
+/* exported getFirstChar */
+
+// input: string
+// output: first character of string
+
+// Create storage for output
+// Check each char to determine if it is the first character
+// if first character or only character
+// add current letter to output
+// return the final output
+
+function getFirstChar(string) {
+  var firstChar = '';
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] === string[0]) {
+      firstChar = string[i];
+    }
+  }
```

Comment:
This should not require a loop. You already know what index holds the first value, you should just retrieve it and return it.

Reply:
Thanks for the feedback, fixed!